bubble-wrap.yaml

```diff
@@ -167,8 +167,8 @@ global:
     sdk_shield_color: [0.506,0.192,0.169]
     sdk_shield_text_color: white
     #
-    interactive_dev: false
-    interactive_prod: true
+    # enable interactivity for key features
+    interactive: true
```

Comment:
originally my thinking was `ux_*` was for user interaction / app level and `sdk_*` for changing parts of the base map. But it seems like `sdk_*` is generally better (and one less name space). So I'd name this `sdk_interactive`.